### PR TITLE
Format Code and remove useless code

### PR DESCRIPTION
**Pull request by IanCao**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

XXXXX

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

File: client/src/main/java/com/alibaba/nacos/client/naming/utils/Chooser.java [hunk 1]
```diff
@@ -138,7 +138,7 @@ public void refresh() {
             }
 
             double doublePrecisionDelta = 0.0001;
-            if (index != 0 && !(Math.abs(weights[index - 1] - 1) < doublePrecisionDelta)) {
+            if (index == 0 || (Math.abs(weights[index - 1] - 1) < doublePrecisionDelta)) {
```
Comment (nkorange) [inline client/src/main/java/com/alibaba/nacos/client/naming/utils/Chooser.java]: Seems it should be ```if (!(index == 0 || (Math.abs(weights[index - 1] - 1) < doublePrecisionDelta)))```

Comment (IanCao) [reply client/src/main/java/com/alibaba/nacos/client/naming/utils/Chooser.java]: yes its my fault i forget return in this part
just according to p3c (! is not clear for code)

File: client/src/main/java/com/alibaba/nacos/client/utils/IPUtil.java [hunk 2]
```diff
@@ -26,31 +26,23 @@
 @SuppressWarnings("PMD.ClassNamingShouldBeCamelRule")
 public class IPUtil {
 
-    public static boolean isIPV4(String addr) {
-        if (null == addr) {
-            return false;
-        }
-        String rexp = "^((25[0-5]|2[0-4]\\d|[01]?\\d\\d?)\\.){3}(25[0-5]|2[0-4]\\d|[01]?\\d\\d?)$";
+    private static final String IPV4_PATTERN = "^((25[0-5]|2[0-4]\\d|[01]?\\d\\d?)\\.){3}(25[0-5]|2[0-4]\\d|[01]?\\d\\d?)$";
```
Comment (kezhenxu94) [inline client/src/main/java/com/alibaba/nacos/client/utils/IPUtil.java]: I would suggest make `IPV4_PATTERN` type of `Pattern`, then you don't need to compile it every time

File: client/src/main/java/com/alibaba/nacos/client/naming/utils/IoUtils.java [hunk 3]
```diff
@@ -178,15 +178,22 @@ public static boolean isGzipStream(byte[] bytes) {
...
-
-        return out.toByteArray();
+        try {
+            gis = new GZIPInputStream(new ByteArrayInputStream(raw));
+            out = new ByteArrayOutputStream();
+            IoUtils.copy(gis, out);
+            return out.toByteArray();
+        } finally {
+            if (out != null) {
+                out.close();
```
[the remark this answers is not in the document]
Comment (IanCao) [reply client/src/main/java/com/alibaba/nacos/client/naming/utils/IoUtils.java]: yes i know but this module build by java6

File: client/src/main/java/com/alibaba/nacos/client/utils/IPUtil.java (hunk 2 above)
Comment (IanCao) [reply client/src/main/java/com/alibaba/nacos/client/utils/IPUtil.java]: make sense

**Pull request merged**